src/main/java/org/reaktivity/nukleus/ws/internal/types/stream/WsFrameFW.java

```diff
@@ -138,17 +171,22 @@ private int lengthSize()
         }
     }
 
-    private int length()
+    private int lengthSize()
     {
-        int length = buffer().getByte(offset() + FIELD_OFFSET_MASK_AND_LENGTH) & 0x7f;
+        return lengthSize(buffer().getByte(offset() + FIELD_OFFSET_MASK_AND_LENGTH));
+    }
+
+    private int length(DirectBuffer buffer, int offset)
```

Comment:
Suggest making static and move to end of file.

Note: will catch bugs below where `offset()` is still used instead of `offset` in some switch cases.